stdlib/int64.mli

```diff
@@ -246,6 +246,44 @@ val max: t -> t -> t
     @since 4.13
  *)
 
+val popcount: t -> int
+(** Hamming weight.
+    [popcount n] is the number of 1 bits in the binary representation of [n].
+    Negative [n] are represented in two's complement. *)
+
+val unsigned_bitsize: t -> int
+(** [unsigned_bitsize n] is the minimal number of bits needed to represent
+    [n] as an unsigned binary number.  It is the smallest integer [i]
+    between 0 and 64 such that [0 <= n < 2{^i}] (unsigned). *)
+
+val signed_bitsize: t -> int
+(** [unsigned_bitsize n] is the minimal number of bits needed to represent
+    [n] as a signed, two's complement binary number.
+    It is the smallest integer [i] between 1 and 64 such that
+    [-2{^i-1} <= n < 2{^i-1}] (signed). *)
+
+val nlz: t -> int
+(** [nlz n] is the number of leading (most significant) 0 bits in
+    the binary representation of [n].  It is an integer between 0 and 64
+    included.  If [n] is negative, [nlz n = 0] since the most significant
```

Comment:
"inclusive" rather than "included"? (also in `nls`) - FWIW you haven't been as explicit in the strings for bitsize.

Reply:
Ah, but your use of “minimal number” taught me that I’ve been incorrectly using “minimum number” for at least 30 years!!